views/pulsar/layouts/base.html.twig

```diff
@@ -58,76 +58,79 @@
     <a class="skip-link" href="#skip-target" tabindex="1">Skip to main content</a>
 
 {% block body %}
-    <div class="container" id="top">
+    <div role="main" class="container" id="top">
```

Comment:
This is no longer correct, `tab__content` is now `role="main"`

Reply:
Clarify this for me, as I have

```
<main class="tab__content" id="skip-target">
```